portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +35,62 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final int MIN_COMMENT_LIMIT = 5;
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    Query query = new Query(Comment.ENTITY_NAME_PARAM).addSort(Comment.TIME_PARAM, SortDirection.DESCENDING);
+    
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List comments = new ArrayList<>();
```

Comment:
Can you add the generic parameter here like: `List<Comment> ...`

Reply:
Done.